### PR TITLE
feat: add Moonbase Testnet

### PR DESCRIPTION
**Pull request by guy93354**
Moonbase Testnet was missing from the testnets list, so I added it by following the steps from https://docs.hyperlane.xyz/docs/deploy-hyperlane

### Description

Add Moonbase Testnet

### Backward compatibility

No

### Testing

Used Hyperlane CLI

### CODE REVIEW

File: chains/moonbase/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://moonbase.moonscan.io/
```
Comment (paulbalaji) [inline chains/moonbase/metadata.yaml:3]:
```suggestion
  - apiUrl: https://api-moonbase.moonscan.io/api
```

File: chains/moonbase/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    family: blockscout
+    name: Moonbase Alpha Testnet Explorer
+    url: https://moonbase.moonscan.io/
```
Comment (paulbalaji) [inline chains/moonbase/metadata.yaml:6]:
```suggestion
    url: https://moonbase.moonscan.io
```

File: chains/moonbase/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://moonbase.moonscan.io/
+    family: blockscout
```
Comment (paulbalaji) [inline chains/moonbase/metadata.yaml:4]:
```suggestion
    family: etherscan
```

File: chains/moonbase/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+chainId: 1287
+isTestnet: true
+name: moonbase
+nativeToken:
+  decimals: 18
+  name: Dev
+  symbol: DEV
+protocol: ethereum
+rpcUrls:
+  - http: https://moonbase-alpha.public.blastapi.io
```
Comment (paulbalaji) [inline chains/moonbase/metadata.yaml:21]: consider adding more RPCs, as found on https://chainlist.org/?search=1287

File: chains/moonbase/logo.svg [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+-8 -161 30 -264 134 -369 29 -29 78 -64 111 -80 51 -24 70 -27 163 -28 96 0
+110 2 160 28 70 36 114 78 152 145 16 28 31 52 33 52 1 0 15 -49 31 -110 l29
+-110 62 0 c35 0 63 3 63 6 0 4 -15 61 -34 128 -19 66 -45 163 -57 215 l-23 94
+23 91 c12 50 38 146 57 212 19 67 34 124 34 128 0 3 -28 6 -63 6 l-63 0 -24
+-97 c-13 -54 -27 -104 -30 -112 -5 -8 -17 5 -37 41 -35 61 -93 117 -155 149
+-62 33 -211 37 -287 9z m212 -116 c60 -20 129 -82 155 -138 48 -101 48 -258 1
+-351 -48 -93 -145 -155 -244 -155 -61 0 -143 33 -187 75 -124 121 -123 390 3
+513 66 64 179 88 272 56z"/>
+</g>
+</svg>
```
Comment (paulbalaji) [inline chains/moonbase/logo.svg:20]: ran through svg minifier https://jakearchibald.github.io/svgomg/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="230.7" height="185.3" version="1.0" viewBox="0 0 173 139"><path d="M65.1 28c-11 4-18.1 10.9-23.9 22.8a33.7 33.7 0 0 0-4 17c-.8 16.1 3 26.4 13.4 36.9 2.9 2.9 7.8 6.4 11.1 8 5.1 2.4 7 2.7 16.3 2.8 9.6 0 11-.2 16-2.8 7-3.6 11.4-7.8 15.2-14.5a34 34 0 0 1 3.3-5.2c.1 0 1.5 4.9 3.1 11l2.9 11h6.2c3.5 0 6.3-.3 6.3-.6 0-.4-1.5-6.1-3.4-12.8-1.9-6.6-4.5-16.3-5.7-21.5l-2.3-9.4 2.3-9.1c1.2-5 3.8-14.6 5.7-21.2a230 230 0 0 0 3.4-12.8c0-.3-2.8-.6-6.3-.6h-6.3l-2.4 9.7c-1.3 5.4-2.7 10.4-3 11.2-.5.8-1.7-.5-3.7-4.1a39.6 39.6 0 0 0-15.5-14.9 45.9 45.9 0 0 0-28.7-.9zm21.2 11.6a30 30 0 0 1 15.5 13.8 45 45 0 0 1 .1 35.1A28.3 28.3 0 0 1 77.5 104c-6.1 0-14.3-3.3-18.7-7.5-12.4-12.1-12.3-39 .3-51.3a27.7 27.7 0 0 1 27.2-5.6z"/></svg>
```

File: chains/moonbase/addresses.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+domainRoutingIsmFactory: "0xA318540392ec7d0809B73B6D2065A50D7C62675B"
```
Comment (paulbalaji) [inline chains/moonbase/addresses.yaml:1]: could you also add the `merkleTreeHook` address please, can find by looking at deployment logs

and if you can't find an `interchainGasPaymaster`, set it to the zero address `0x0000000000000000000000000000000000000000`

**Review by guy93354 — COMMENTED** [review]
All done, thanks Paul for the time and feedback.

File: chains/moonbase/metadata.yaml [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  - http: https://moonbase.unitedbloc.com
+  - http: https://rpc.testnet.moonbeam.network
+  - http: https://rpc.testnet.moonbeam.network
+
```
Comment (paulbalaji) [inline chains/moonbase/metadata.yaml:25]:
```suggestion
  - http: https://rpc.testnet.moonbeam.network
```

File: chains/moonbase/metadata.yaml [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  - apiUrl: https://api-moonbase.moonscan.io/api
+    family: etherscan
+    name: Moonbase Alpha Testnet Explorer
+    url: https://moonbase.moonscan.io
+chainId: 1287
+deployer:
+  name: guy93354
+  url: https://github.com/guy93354
+displayName: Moonbase Testnet
+chainId: 1287
```
Comment (paulbalaji) [inline chains/moonbase/metadata.yaml:12]:
```suggestion
```

**Pull request merged**